src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1000,6 +1018,25 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                    call mpas_pool_get_field(scratchPool, 'btrvel_temp', btrvel_tempField)
                    btrvel_temp => btrvel_tempField % array
 
+                   ! Subtract tidal potential from ssh, if needed
+                   if (config_use_tidal_potential_forcing) then
+                     call mpas_pool_get_array(statePool, 'sshSubcycle', sshSubcycleCurNoTides, oldBtrSubcycleTime)
+                     call mpas_pool_get_array(statePool, 'sshSubcycle', sshSubcycleNewNoTides, newBtrSubcycleTime)
+                     call mpas_pool_get_array(forcingPool,'sshSubcycleCurWithTides', sshSubcycleCur)
+                     call mpas_pool_get_array(forcingPool,'sshSubcycleNewWithTides', sshSubcycleNew)
```

Comment:
Same comment here, please consider making the local variable names the same as those in the pool.

Reply:
I took care of this the best I could in my latest commits.